HLTrigger/Configuration/test/test_hltFindDuplicates.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+# Pass in name and status
+function die {
+  printf "\n%s: status %s\n" "$1" "$2"
+  if [ $# -gt 2 ]; then
+    printf "%s\n" "=== Log File =========="
+    cat $3
+    printf "%s\n" "=== End of Log File ==="
+  fi
+  exit $2
+}
+
+if [ -z "${SCRAM_TEST_PATH}" ]; then
+  printf "\n%s\n\n" "ERROR -- environment variable SCRAM_TEST_PATH not defined"
+  exit 1
+fi
+
+# run test job
+TEST_MENUS=(
+#  Fake
+#  Fake1
+#  Fake2
+#  FULL
+#  GRun
+#  HIon
+  PRef
```

Comment:
why `PRef` in particular?

Reply:
From the description :)

> In order to make the unit test fast (a few seconds), the test currently uses only the PRef menu (the simplest menu in the release with some duplicate modules).

Reply:
how long does the `GRun` takes? IMHO that would be more useful to test.

Reply:
`GRun` takes ~270 seconds, PRef takes less than 5 seconds (and all the menus except `GRun` and the combined table take ~10 seconds in total). Of course, I'm happy to update the test using the menus of your choice.